.github/workflows/_check_clang_static_analyzer.yml

```diff
@@ -3,11 +3,11 @@ name: Run Clang Static Analyzer
 on:
   workflow_call:
     inputs:
-      ledger-app-workflows_ref:
+      workflows_ref:
```

Comment:
Hmm not sure I like this rename
In the context of reusable workflows, "workflows_ref" is not very explicit. "ledger-app-workflows_ref" we understand at first glance that what it is